suite-common/icons/src/config.ts

```diff
@@ -0,0 +1,51 @@
+import { TokenAddress } from 'suite-common/wallet-types/src';
+
+import { Color } from '@trezor/theme';
+
+import { IconName, FlagIconName, CryptoIconName } from './icons';
+
+export interface IconProps {
+    name: IconName;
```

Comment:
`name` is a valid HTML property used in forms so there is some ambiguity. We used to call this prop `icon`. But feel free to disregard this nitpick...

Reply:
it was `name` already and I think it makes sense – `<Icon/>` having a `name` prop